django/db/backends/oracle/base.py

```diff
@@ -334,13 +385,21 @@ def check_constraints(self, table_names=None):
             cursor.execute("SET CONSTRAINTS ALL DEFERRED")
 
     def is_usable(self):
+        if self.connection is None:
+            return False
```

Comment:
This change looks unrelated :thinking:

Reply:
You are right it seems not useful with the current changes. I have removed this.